docs/content/guides/getting-started/introduction.md

```diff
@@ -8,62 +8,118 @@ react:
   metaTitle: Introduction - React Data Grid | Handsontable
 ---
 
-# Handsontable guides
+# Handsontable documentation
 
 [[toc]]
 
-Welcome to the Handsontable guides. 👋
+::: only-for javascript
+::: tip
+Using React? [Switch to the React version of this documentation](../react-data-grid).
```

Comment:
Shouldn't the link to React be `../../react-data-grid`? The current one gives me 404

Reply:
@warpech – hm, on my side, it's the other way round: `../../react-data-grid` (resolves as `http://localhost:8080/docs/react-data-grid`) gives me 404, and `../react-data-grid` (resolves as `http://localhost:8080/docs/next/react-data-grid`) works.

Reply:
Got it, fixed: [bfcaa22](https://github.com/handsontable/handsontable/pull/9862/commits/bfcaa22cf8f29e4c394b3a8b271535b5e4b07bfa)